addons/point_of_sale/tools/posbox/configuration/posbox_update.sh

```diff
@@ -21,6 +21,8 @@ if [ -d /home/pi/odoo/addons/point_of_sale/tools/posbox/overwrite_after_init ];
     rm -r /home/pi/odoo/addons/point_of_sale/tools/posbox/overwrite_after_init
 fi
 
+pip3 install -r /home/pi/odoo/addons/point_of_sale/tools/posbox/configuration/requirements.txt --user --break-system-package
```

Comment:
It's in master, it can only occur if users downgrade their database, does it really happen?

Reply:
It could happen if an existing customer decides to start using IoT and hasn't upgraded to latest version yet. But yes it's a rare situation.
Also I've realised it only needs a restart to fix so it's probably okay to leave it as is